src/rp2040/hardware_regs/RP2040.svd

```diff
@@ -32214,7 +32214,8 @@ SPDX-License-Identifier: BSD-3-Clause
 
                             After writing, this register must be polled until it returns all-zero. Until this point, it is unsafe to restart the channel.</description>
                         <bitRange>[15:0]</bitRange>
-                        <access>write-only</access>
+                        <access>read-write</access>
+                        <modifiedWriteValues>clear</modifiedWriteValues>
```

Comment:
https://datasheets.raspberrypi.com/rp2040/rp2040-datasheet.pdf and https://datasheets.raspberrypi.com/rp2350/rp2350-datasheet.pdf both document "DMA: CHAN_ABORT" as being Type `SC`. So out of curiosity, why does `RP2040.svd` use `clear` but `RP2350.svd` uses `oneToClear` ?

Reply:
good catch